src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVIntervalTree.java

```diff
@@ -0,0 +1,904 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
```

Comment:
@lbergelson Yes, very similar functionality, except...
The original is an interval tree on a single integer number line, we need an interval tree with a number line for each contig.  Also, the one in htsjdk has been hacked from the original to use fully closed intervals rather than half-open.  Since we sometimes need to mark locations as 0-length intervals, that's not helpful.
Too bad I didn't make the interval concept an interface when I first wrote it.  Probably no way to reassemble Humpty Dumpty at this point.